src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -3881,6 +3877,8 @@ def test_sql_zone_resilient_pool(self, resource_group, resource_group_location,
                      JMESPathCheck('dtu', 250),
                      JMESPathCheck('zoneRedundant', True)])
 
+    # Skipping test because provisioning of servers are not available on euap region, but feature is only on euap region
```

Comment:
How can feature be available in euap region without server available?

Reply:
changed api version in yaml files per you suggestion